lib/src/value/color.dart

```diff
@@ -156,11 +156,13 @@ class SassColor extends Value {
     switch (space) {
       case ColorSpace.lab:
       case ColorSpace.oklab:
-        return fuzzyEquals(channel0, 0);
+        return fuzzyEquals(channel0, 0) || fuzzyEquals(channel0, 100);
 
       case ColorSpace.lch:
       case ColorSpace.oklch:
-        return fuzzyEquals(channel0, 0) || fuzzyEquals(channel1, 0);
+        return fuzzyEquals(channel0, 0) ||
+            fuzzyEquals(channel0, 100) ||
+            fuzzyEquals(channel1, 0);
```

Comment:
Being unfamiliar with channels and power, should `channel1` be compared against 100 too?

Reply:
Nitpicking a bit: I'm confused by the function called `isChannel2Powerless` where `channel0` and `channel1` are checked, but not `channel2`. I'm assuming that's intended, but I don't understand why xD

Reply:
See https://github.com/sass/sass/blob/main/proposal/color-4-new-spaces.md#powerless-components. For example, for an LCH color, the hue is powerless if the chroma is zero because the color is grayscale and so hue doesn't matter anymore.